interop/fake_grpclb/fake_grpclb.go

```diff
@@ -0,0 +1,178 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Runs a fake grpclb balancer server, can be useful for testing.
+// The name of the service to load balance for and the addresses
+// of that service are provided by command line flags.
+package main
+
+import (
+	"flag"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	durationpb "github.com/golang/protobuf/ptypes/duration"
+	"google.golang.org/grpc"
+	lbpb "google.golang.org/grpc/balancer/grpclb/grpc_lb_v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/alts"
+	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/status"
+	"google.golang.org/grpc/testdata"
+)
+
+var (
+	port        = flag.Int("port", 10000, "Port to listen on.")
+	backendIps  = flag.String("backend_addrs", "", "Comma separated list of backend IP/port addresses.")
+	useALTS     = flag.Bool("use_alts", false, "Listen on ALTS credentials.")
```

Comment:
Merge this and TLS flag into one that takes a string?
Otherwise we will need to deal with `*alts==true && *tls==true`, and it's hard to add another one (insecure maybe), just like in interop client.

Reply:
hmm, I agree this is a little weird, but the current flags do allow specifying insecure (by keeping both of these flags off), and it's convenient to be able to pass the same flags to this server as for the interop server

What do you think?